kubernetes/deployment.yaml

```diff
@@ -21,10 +21,10 @@ spec:
             httpHeaders:
             - name: Accept
               value: text/plain
-            path: /health
+            path: /health/zwave
```

Comment:
`/health` api is more generic, it also checks for mqtt status (if it's not disabled)